exercism/hamming/hamming.js

```diff
@@ -1,23 +1,14 @@
 var Hamming = function() {};
 
 Hamming.prototype.checkForSameLength = function(firstStrand, secondStrand) {
-  if (firstStrand.length === secondStrand.length) {
-    return true;
-  }
-  return false;
+  return firstStrand.length === secondStrand.length ? true : false
 }
 
 Hamming.prototype.compute = function(firstStrand, secondStrand) {
   if (this.checkForSameLength(firstStrand, secondStrand)) {
-    let firstArray = firstStrand.split('');
-    let secondArray = secondStrand.split('');
-    let hammingDistance = 0
-    for (var i = 0; i < firstArray.length; i++) {
-      if (firstArray[i] !== secondArray[i]) {
-        hammingDistance++
-      }
-    }
-    return hammingDistance
+    return firstStrand.split('').reduce( function(accumulator, current, index){
```

Comment:
I am really liking your approach. Reduce is going to be with us for quite a while in JavaScript.